Makefile

```
@@ -461,6 +462,23 @@ build-ova-proxy-image: check_container_runtime
 push-ova-proxy-image: build-ova-proxy-image
 	$(CONTAINER_CMD) push $(OVA_PROXY_IMAGE)$(PLATFORM_SUFFIX)
 
+build-deep-inspection-image: check_container_runtime ## Build forklift-deep-inspection (virt-inspector + VDDK) container image
```

checkmake (0.3.2)

[warning] Target body for "build-deep-inspection-image" exceeds allowed length of 5 lines (7).

(maxbodylength)